docs/main/05-reference/02-clusters/01-clusters.md

```diff
@@ -18,15 +20,60 @@ metadata:
   labels:
     environment: dev
 spec:
-  # Cluster identifier: required
-  id: cluster-id
-  # Path to be used by Kratix to create the buckets
-  bucketPath: bucket-path
+  # Cluster identifier: optional, appended path to be used within the StateStore
+  path: path/in/statestore
+  # Required
+  stateStoreRef:
+    # The name of the StateStore to use: required
+    name: default
+    # The namespace of the StateStore to use: optional, defaults to the clusters namespace
+    namespace: default
+    # The kind of the StateStore to use: required, valid options: GitStateStore, BucketStateStore
+    kind: BucketStateStore
 ```
 
 When a new Cluster is registered in the Platform cluster (i.e., a new Cluster Resource is
-created), Kratix will create a pair of buckets in the [Repository](../01-deployment-topology/01-deployment-topology.md#gitops-repository): one for `resources`, one for `crds`. The
-full path of the buckets will be:
+created), Kratix will write to two paths in the [StateStore](../06-statestore/01-statestore.md):
+one for `resources`, one for `crds`. The path within the `StateStore` follows the following pattern:
+```
+<statestore.Spec.Path>/<cluster.Spec.Path>/<cluster.Metadata.Namespace>/<cluster.Metadata.Name>/
```

Comment:
this is a bit hard to parse...

suggestion:
```
statestore.Spec.Path/
    cluster.Spec.Path/
        cluster.Metadata.Namespace/
            cluster.Metadata.Name/
```

Reply:
yeah I struggled to write this out cleanly. I think thats an improvement, but its still not great!